cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -214,4 +219,48 @@ protected void addService(Service service) {
   protected void addService(String name, HttpServiceHandler handler, HttpServiceHandler...handlers) {
     configurer.addService(new BasicService(name, handler, handlers));
   }
+
+  /**
+   * Schedules the specified {@link Workflow}
+   * @param schedule the schedule to be added for the Workflow
+   * @param workflowName the name of the Workflow
+   */
+  protected void scheduleWorkflow(Schedule schedule, String workflowName) {
+    scheduleWorkflow(schedule, workflowName, new HashMap<String, String>());
```

Comment:
Prefer never to use new HashMap<...>() directly, better use Maps.newHashMap().
But here I would use Collections.emptyMap() or ImmutableMap.of().